contracts/lib/LibPerpetuals.sol

```diff
@@ -86,9 +86,10 @@ library Perpetuals {
         uint256 deleveragingCliff,
         uint256 insurancePoolSwitchStage
     ) internal pure returns (uint256) {
-        if (poolTarget == 0) {
+        if (deleveragingCliff == insurancePoolSwitchStage || poolTarget == 0) {
```

Comment:
This case will never occur in our protocol since deleveragingCliff must be set higher than insurancePoolSwitchStage.

However, it has been included as a special case in this function in case other implementations of tracer do not have the same requirement. It might be better to change this to a require() instead of special case.

Reply:
should this be `deleveragingCliff >= insurancePoolSwitchStage`?

Reply:
ok sounds good